tests/ut/ir/statements/test_split_aiv_scope_stmt.py

```
@@ -0,0 +1,88 @@
+    assert isinstance(scope, ir.ScopeStmt)
+    assert scope.scope_kind == ir.ScopeKind.SplitAiv
+    assert scope.split == ir.SplitMode.UP_DOWN
+    assert scope.count == 2
+    assert isinstance(scope.body, ir.AssignStmt)
+
+
+def test_construct_rejects_none_mode():
+    """The ctor INTERNAL_CHECK rejects SplitMode.NONE (a no-op AIV split)."""
+    span = ir.Span("test.py", 1, 1, 1, 10)
+    body = _make_body(span, *_vars(span))
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)